esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -105,7 +105,7 @@ IPropertyTree * fetchESDLBindingFromDali(const char *process, const char *bindin
     Owned<IRemoteConnection> conn;
     try
     {
-        conn.set(querySDS().connect(ESDL_BINDINGS_ROOT_PATH, myProcessSession(), RTM_LOCK_READ|RTM_CREATE_QUERY, SDS_LOCK_TIMEOUT_DESDL));
+        conn.set(querySDS().connect(ESDL_BINDINGS_ROOT_PATH, myProcessSession(), RTM_LOCK_READ | RTM_CREATE_QUERY, SDS_LOCK_TIMEOUT_DESDL));
         if (!conn)
         {
             DBGLOG("Unable to connect to ESDL Service binding information in dali %s", ESDL_BINDINGS_ROOT_PATH);
```

Comment:
@jakesmith I'm glad you commented on this, I believe I encountered issues w/ SDS locks and found that closing the connection immediately (even before querying the root) alleviated the lock contention, the scary part is that querying the root after the close always works. 

I will follow up via this issue: https://track.hpccsystems.com/browse/HPCC-16707